snapcraft.yaml

```diff
@@ -8,6 +8,7 @@ confinement: strict
 type: base
 build-base: core22
 grade: stable
+assumes: [snapd-2.55.5]
```

Comment:
the syntax is wrong: afaik it's snapd2.55.5 without -

Reply:
Uh, silly me - thanks for catching this. You are correct and I fixed it now.